hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/HazelcastClientNotActiveException.java

```diff
@@ -0,0 +1,16 @@
+package com.hazelcast.client.spi.impl;
+
+/**
+ * Thrown when Hazelcast client is not active during an invocation.
+ */
+public class HazelcastClientNotActiveException extends IllegalStateException {
```

Comment:
is this exception is thrown to the caller? if so, `impl` package is wrong place...